### DISCUSSION TITLE
Can anyone provide guidance on "Collection Error: variables attribute not supported"?

### DISCUSSION THREAD

**Discussion by dukejansen** [meisnate12/Plex-Meta-Manager#1156] [Q&A] [answered]
I'm seeing this error all throughout my logs when trying to use a variety of the "standard" PMM configuration files under https://github.com/meisnate12/Plex-Meta-Manager-Configs/tree/master/PMM

Specifically:
https://github.com/meisnate12/Plex-Meta-Manager-Configs/blob/master/PMM/movie/universe.yml
and also some of the ones under
https://github.com/meisnate12/Plex-Meta-Manager-Configs/tree/master/PMM/chart

I'm using the latest PMM release, via docker, but not a developer branch, just the main release.

I'm pulling the PMM files directly from github, via my own config.

Is this a mismatch between the PMM configuration files in github (latest?) and the version of PMM I'm using (release)?

Is this a bug in some of these 'standard' PMM configuration files? Are they stale? Trying to figure out what could be going on here, whether it's a known issue or a new bug or user error or what.

**Comment by chazlarson** [comment, marked as answer]
This is a known issue that is fixed in the develop branch.
